assets/models/system/compvis-stable-diffusion-v1-4/spec.yaml

```diff
@@ -29,4 +29,4 @@ tags:
       Standard_ND96amsr_A100_v4,
       Standard_ND96asr_v4
     ]
-version: 5
\ No newline at end of file
+version: 6
```

Comment:
Version update is not just for description?

Reply:
updated.